### PR TITLE
Added documentation for multi-line boolean statements

### PR DESCRIPTION
**Pull request by adamjubert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2131,6 +2131,39 @@ no parameters.
   # good
   array.reverse_each { ... }
   ```
+* <a name="multi-line-boolean"></a>
+
+<sup>[[link](#multi-line-boolean)]</sup>
```
Comment (jkolber) [inline README.md]: I think this link should come after the blurb you're linking to. See the section above (lines 2119-2125).

File: README.md [hunk 2]
```diff
@@ -2131,6 +2131,39 @@ no parameters.
...
+  ```
+
+  If only two booleans, compare them on the same line.
+  ```ruby
+  # good
+  if some_boolean_one? || some_boolean_two?
+    ...
+    ...
+  end
+  ```
```
Comment (jkolber) [inline README.md]: Lines 2138-2165 should be indented two spaces to match the rest of the document.

Comment (adamjubert) [reply README.md]: Aren't they already?

File: README.md (hunk 1 above)
Comment (adamjubert) [reply README.md]: Done

**Pull request merged**